include/seqan/seeds/seeds_extension.h

```diff
@@ -796,7 +796,7 @@ extendSeed(Seed<Simple, TConfig> & seed,
     // SEQAN_ASSERT_LT(scoreMismatch(scoringScheme), 0);
     SEQAN_ASSERT_LT(scoreGapOpen(scoringScheme), 0);
     SEQAN_ASSERT_LT(scoreGapExtend(scoringScheme), 0);
-    SEQAN_ASSERT_EQ(scoreGapExtend(scoringScheme), scoreGapOpen(scoringScheme));
+    //SEQAN_ASSERT_EQ(scoreGapExtend(scoringScheme), scoreGapOpen(scoringScheme));
```

Comment:
OK. I found that we need an investigation about this.
So I do not change the source code and make another issue for this.